docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -196,11 +336,242 @@ To process the received command on the server side:
+
+   ret = mc_server_foreach_client(g_mc_server, activated_client_cb, &client_list);
+   ```
+
+3. Send the event to the client using the corresponding `mc_server_send_custom_event()`. Use the client name retrieved in the previous step to identify the client.
+For example, to send a your own event to the client, use the `mc_server_send_custom_event()` with the event in the third parameter:
+
+   ```
+   char *request_id = NULL;
+
+   ret = mc_server_send_custom_event(g_server_h, client_name, "evnet1", NULL, &request_id);
+   ```
+
+4. Destroy the media controller server handle using the `mc_server_destroy()`, when media controller server handle is no longer needed:
+
+   ```
+   mc_server_destroy(g_server_h);
+   ```
+
+To process the received event on the client side:
+
+1. Create the media controller client handle using the `mc_client_create()`:
+
+   ```
+   ret = mc_client_create(&g_client_h);
+   ```
+
+2. Define the callback that is invoked when the client receives the event.
+
+   For example, to define a callback for a custom event:
+
+   ```
+   void
+   event_received_cb(const char* server_name, const char *request_id, const char *event, bundle *data, void *user_data)
+   {
+       dlog_print(DLOG_DEBUG, LOG_TAG, "Server Name: %s, Request id: %s, Event: %s\n", server_name, request_id, event);
+   }
+   ```
+
+3. Register the callback:
+
+   - To register a callback for a custom events, use the `mc_client_set_custom_event_received_cb()`.
+
+      For example, to register a callback for a custom event:
+
+      ```
+      ret = mc_client_set_custom_event_received_cb(g_client_h, event_received_cb, NULL);
+      ```
+
+4. Destroy the media controller client handle using the `mc_client_destroy()`, when media controller client handle is no longer needed:
+
+   ```
+   mc_client_destroy(g_client_h);
+   ```
+
+
+<a name="send_event_reply"></a>
+To send the reply of completed custom event on the client side:
+
+1. Send the reply of completed custom event using the `mc_client_send_event_reply()` with the request id of the custom event in the third parameter and the result of the custom event in fourth parameter:
+
+   ```
+   int result_code = 0;
+
+   ret = mc_client_send_event_reply(g_server_h, server_name, request_id, result_code, NULL);
+   ```
+
+To receive the reply of processing command on the server side:
+
+1. Define the callback that is invoked when the server receives the reply:
+
+   ```
+   void
+   event_reply_received_cb(const char *client_name, const char *request_id, int result_code, bundle *data, void *user_data)
+   {
+       dlog_print(DLOG_DEBUG, LOG_TAG, "Client Name: %s, Request Id: %s, Result Code: %d\n", client_name, request_id, result_code);
+   }
+   ```
+
+2. Register the callback:
+
+   ```
+   ret = mc_server_set_event_reply_received_cb(g_server_h, event_reply_received_cb, NULL);
+   ```
+
+> **Note**
+>
+> This feature supports Tizen 4.0 and higher for Mobile.
+
 <a name="serverstate"></a>
```

Comment:
As mentioned earlier, please avoid using HTML code. Use only MD specific code.

Reply:
The HTML tag is not used. So I removed it.